static/app/views/settings/organizationMembers/inviteBanner.spec.tsx

```diff
@@ -0,0 +1,173 @@
+import {browserHistory} from 'react-router';
+import moment from 'moment';
+
+import {act, render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {MissingMember} from 'sentry/types';
+import {DEFAULT_SNOOZE_PROMPT_DAYS} from 'sentry/utils/promptIsDismissed';
+import {InviteBanner} from 'sentry/views/settings/organizationMembers/inviteBanner';
+
+const missingMembers = {
+  integration: 'github',
+  users: TestStubs.MissingMembers() as MissingMember[],
+};
+
+const noMissingMembers = {
+  integration: 'github',
+  users: [] as MissingMember[],
+};
+
+describe('inviteBanner', function () {
+  beforeEach(function () {
+    MockApiClient.clearMockResponses();
```

Comment:
@scttcper do we still need these?

Reply:
technically no, but its fine for now. we can go in and remove them all at once